src/libServer/LookupServer.cpp

```diff
@@ -1423,12 +1430,115 @@ Json::Value LookupServer::GetEthSyncing() {
   return Json::Value(false);
 }
 
-Json::Value LookupServer::GetEthAccounts() {
+Json::Value LookupServer::GetEmptyResponse() {
   LOG_MARKER();
   const Json::Value expectedResponse = Json::arrayValue;
   return expectedResponse;
 }
 
+Json::Value LookupServer::GetEthStorageAt(std::string const& address,
+                                          std::string const& position,
+                                          std::string const& /*blockNum*/) {
+  LOG_MARKER();
+
+  string vname{};
```

Comment:
Nitpick: could these vars be moved closer to the relevant scope?

Reply:
👍